filters.json.example

```diff
@@ -426,11 +426,13 @@
     },
     "eggs":{
         "enabled":"True",
+        "gymname_contains":["sprint","level badge","phone skin","starbucks"],
         "min_level": "1",
         "max_level": "5"
     },
     "raids":{
         "enabled":"True",
+        "gymname_contains":["sprint","level badge","phone skin","starbucks"],
```

Comment:
Same as above.

Reply:
'gymname_contains' is the verbiage.  can you clarify what you are asking?

Reply:
Nevermind, I misread. You are good.